logstash/README.md

```diff
@@ -129,6 +129,7 @@ using `http.host: 127.0.0.1`, default probes should be disabled or overrided
 | `replicas`                | Kubernetes replica count for the StatefulSet (i.e. how many pods)                                                                                                                                                                    | `1`                                   |
 | `resources`               | Allows you to set the [resources][] for the StatefulSet                                                                                                                                                                              | see [values.yaml][]                   |
 | `schedulerName`           | Name of the [alternate scheduler][]                                                                                                                                                                                                  | `""`                                  |
+| `secretCreate`            | Allows you easily create a secret from as variables and from file. For add secrets from file, add suffix `.filepath` to the key of secret key. Useful for store certificates and other secrets. See [values.yaml][] for an example   | `[]`                                  |
```

Comment:
It's probably worth adding a note here to indicate that the `value` will be `base64` encoded on creation, to avoid users putting a `base64` encoded value here and double-encoding.

I'm also not totally keen on the naming of this parameter. 
Maybe just `secrets`?

@jmlrt Thoughts?

Reply:
Thank you for your comments, I'll correct the inaccuracies today.